code/modules/mob/living/blood.dm

```diff
@@ -254,6 +254,11 @@
 
 	var/list/temp_blood_DNA
 	if(small_drip)
+
+		if(T.liquids)
+			var/list/blood_drop = list(/datum/reagent/blood = 0.1) //lower this bad boy
```

Comment:
Comment says to lower it. Did you?

Reply:
yea from 0.5 to 0.1 forgot to remove the comment